zproject/prod_settings_template.py

```diff
@@ -264,6 +264,30 @@
 #    ]
 # }
 
+## These settings can be used to synchronize ldap user groups to Zulip user groups
+## The name of the groups have to be exactly the same!
+## `cn` value of the ldap group is used
+
+## make sure to set `AUTH_LDAP_BIND_DN`, otherwise Zulip cannot access the groups
+# AUTH_LDAP_GROUP_SEARCH = LDAPSearch(
+#     "ou=groups,dc=www,dc=example,dc=com", ldap.SCOPE_SUBTREE, "(objectClass=groupOfUniqueNames)"
+# )
+
+# LDAP_SYNC_GROUPS = True
+
+# LDAP_GROUP_SYNC_LIST = [
+#     "group1",
+#     "group2",
+# ]
+
+## 2 is in most cases the correct realm id, use `manage.py list_realms` to check!
+# LDAP_GROUP_SYNC_REALM_ID = 2
```

Comment:
I imagine we'll end up with some kind of design where we have a dict of dicts, where `d["<subdomain>"]` is a dict of `<ldap_group_name>: <zulip_group_name>` mapping (a bit like `SOCIAL_AUTH_SYNC_CUSTOM_ATTRS_DICT`), but that will need a discussion - so for this prototype PR we can leave this settings design the way you made it

Reply:
I was reluctant to implement something like that since it would be best for this to be UI configurable. Having to migrate the settings from `settings.py` to the UI would have been an additional implementation and also administration step. This could be avoided by directly implementing the mapping into a UI (which is something I unfortunately do not have the time to do it myself).